implementation/contracts/system/TBTCSystem.sol

```diff
@@ -35,7 +35,7 @@ contract TBTCSystem is Ownable, ITBTCSystem, DepositLog {
 
     bool _initialized = false;
     uint256 pausedTimestamp;
-    uint256 pausedDuration = 10 days;
+    uint256 constant pausedDuration = 10 days;
```

Comment:
Thoughts on moving it to `TBTCConstants` library?

Reply:
>  moving it to TBTCConstants library

Leaning towards no. It's not really a TBTCConstant, it's a System constant that will only really be used once.